google_symptoms/delphi_google_symptoms/geo.py

```diff
@@ -37,7 +37,7 @@ def generate_transition_matrix(geo_res):
     if geo_res == "hrr":
         map_df["population"] = map_df["population"] *  map_df["weight"]
 
-    aggregated_pop = map_df.groupby(geo_res).sum().reset_index()
+    aggregated_pop = map_df.groupby(geo_res).sum(numeric_only=True).reset_index()
```

Comment:
I changed this because I'm getting a warning about to-be-deprecated behavior. Thought I'd just throw it in here. 

```
FutureWarning: The default value of numeric_only in DataFrameGroupBy.sum is deprecated. 
In a future version, numeric_only will default to False. Either specify numeric_only or select 
only columns which should be valid for the function.
```

Reply:
The current default does actually seem to be `numeric_only=True`

Reply:
Ah thanks for that! Curious why they're moving away from `numeric_only=True`.